### DISCUSSION TITLE
EJBCA CE 8.2.0.1

### DISCUSSION THREAD

**Discussion by hesunmark** [Keyfactor/ejbca-ce#435] [Announcements]
**Release highlights:** 

- Extended REST interface in EJBCA Community

**For more information:** 

Read the full [EJBCA Community 8.2 Release Notes](https://doc.primekey.com/ejbca/ejbca-release-information/ejbca-release-notes/ejbca-community-8-2-release-notes)

Visit the EJBCA project on [ejbca.org](https://www.ejbca.org/)

<hr /><em>This discussion was created from the release <a href='https://github.com/Keyfactor/ejbca-ce/releases/tag/EJBCA_8_2_0_1_20231206'>EJBCA CE 8.2.0.1</a>.</em>

**Comment by KarolinHem** [comment]
And you can go check out the release highlights in this video: [What's New in EJBCA 8.2 Community edition](https://www.youtube.com/watch?v=MAueccZwk9s)
